fabric/src/main/java/dev/amsam0/voicechatdiscord/FabricPlatform.java

```diff
@@ -170,17 +170,17 @@ private Text toNative(Component component) {
 
             var clickEvent = component.clickEvent();
             if (clickEvent != null) {
-                ClickEvent.Action action = null;
+                net.minecraft.text.ClickEvent.Action action = null;
                 switch (clickEvent.action()) {
-                    case OPEN_URL -> action = ClickEvent.Action.OPEN_URL;
-                    case OPEN_FILE -> action = ClickEvent.Action.OPEN_FILE;
-                    case RUN_COMMAND -> action = ClickEvent.Action.RUN_COMMAND;
-                    case SUGGEST_COMMAND -> action = ClickEvent.Action.SUGGEST_COMMAND;
-                    case CHANGE_PAGE -> action = ClickEvent.Action.CHANGE_PAGE;
-                    case COPY_TO_CLIPBOARD -> action = ClickEvent.Action.COPY_TO_CLIPBOARD;
+                    case OPEN_URL -> action = net.minecraft.text.ClickEvent.Action.OPEN_URL;
+                    case OPEN_FILE -> action = net.minecraft.text.ClickEvent.Action.OPEN_FILE;
+                    case RUN_COMMAND -> action = net.minecraft.text.ClickEvent.Action.RUN_COMMAND;
+                    case SUGGEST_COMMAND -> action = net.minecraft.text.ClickEvent.Action.SUGGEST_COMMAND;
+                    case CHANGE_PAGE -> action = net.minecraft.text.ClickEvent.Action.CHANGE_PAGE;
+                    case COPY_TO_CLIPBOARD -> action = net.minecraft.text.ClickEvent.Action.COPY_TO_CLIPBOARD;
                     default -> warn("Unknown click event action: " + clickEvent.action());
                 }
-                style = style.withClickEvent(new ClickEvent(action, clickEvent.value()));
+                style = style.withClickEvent(new net.minecraft.text.ClickEvent(action, clickEvent.value()));
```

Comment:
I don't see why we need all the net.minecraft

Reply:
this was auto generated by the fabric updating tool
https://fabricmc.net/wiki/tutorial:migratemappings